sys/sys/memrange.h

```diff
@@ -69,6 +71,16 @@ struct mem_livedump_arg {
 
 #define	MEM_KERNELDUMP	_IOW('m', 53, struct mem_livedump_arg)
 
+#if __has_feature(capabilities)
```

Comment:
Hm, unlike struct capreg, this can totally be used from a non-CHERI binary

Reply:
That's true I guess.  The implementation needs to be restricted to CHERI architectures, but I guess not the structure or ioctl definition itself.